configure.ac

```diff
@@ -0,0 +1,42 @@
+m4_define([version], m4_normalize(m4_esyscmd([git describe --exact HEAD 2>/dev/null || git rev-parse --short HEAD])))
+AC_INIT([mender], [version], [mender@discoursemail.com])
+AC_CANONICAL_TARGET
+AM_INIT_AUTOMAKE([-Wall -Werror foreign])
+AM_SILENT_RULES([yes])
+
+dnl Use the m4 directory for m4 macros (convention):
+AC_CONFIG_MACRO_DIR([m4])
+
+AM_PROG_AR
+LT_INIT([shared])
```

Comment:
I'd drop `([shared])` here. We don't build any shared libraries (yet).

Reply:
Thanks :memo: